package.json

```diff
@@ -2,11 +2,16 @@
   "name": "react-formbuilder",
   "version": "0.0.1",
   "description": "A relatively easy to use form builder (single or multipage), with error reporting and lots of room for customization ",
-  "main": "index.js",
+  "main": "dist/react-formbuilder.js",
   "directories": {
     "test": "test"
```

Comment:
@Pomax after reading https://docs.npmjs.com/files/package.json#directories I still don't get what `directories` in `package.json` is for. Are we supposed to put all top level dirs there?